yt_dlp/extractor/digitalconcerthall.py

```diff
@@ -0,0 +1,201 @@
+                    m3u8_url, video_id, 'mp4', 'm3u8_native', fatal=False)
+                self._sort_formats(formats)
+
+                if item.get('is_interview') == 1:
+                    title = "Interview - " + item.get('title', "unknown interview title")
+                else:
+                    title = (item.get('name_composer') if item.get('name_composer')
+                            else 'unknown composer') + ' - ' + item.get('title', "unknown title")
+                key = item.get('id')
+
+                duration = item.get('duration_total')
+                # avoid filenames that exceed filesystem limits
+                title = (title[:MAX_TITLE_LENGTH] + '..') if len(title) > MAX_TITLE_LENGTH else title
+                # append the duration in minutes to the title
+                title = title + " (" + str(round(duration / 60)) + " min.)"
+                self.debug_out("title: " + title)
+                timestamp = item.get('date').get('published')
+                entries.append({
+                    'id': key,
+                    'title': title,
+                    'url': m3u8_url,
+                    'formats': formats,
+                    'duration': duration,
+                    'timestamp': timestamp,
+                })
+                # use playlist description for video description by default
+                # but if the video has a description, use it
+                description = test_dict.get('short_description') or item.get('short_description')
+                if description:
+                    entries[-1]['description'] = description
+                if item.get('cuepoints'):
+                    chapters = []
+                    first_chapter = 1
+                    for chapter in item.get('cuepoints'):
+                        start_time = chapter.get('time')
+                        # Often, the first chapter does not start at zero.  In this case,
+                        # insert an intro chapter so that first chapter is the start of the music
+                        if (first_chapter == 1) and (start_time != 0):
+                            chapters.append({
+                                'start_time': 0,
+                                'end_time': start_time,
+                                'title': '0. Intro'
+                            })
+                        first_chapter = 0
+                        end_time = start_time + chapter.get('duration')
+                        chapter_title = chapter.get('text')
+                        chapters.append({
+                            'start_time': start_time,
+                            'end_time': end_time,
+                            'title': chapter_title
+                        })
+                    entries[-1]['chapters'] = chapters
+
+        return {
+            '_type': 'playlist',
+            'id': video_id,
+            'title': playlist_title,
+            'entries': entries,
```

Comment:
entries should actually be a paged list

Reply:
per discord conversation, pukkandan will take care of this
https://discord.com/channels/807245652072857610/807246463808700487/918283795923365959